doc/cli.markdown

```diff
@@ -643,39 +642,6 @@ confirm non interactively
 
 device
 
-## env add &#60;key&#62; [value]
```

Comment:
There is a 'Review actions' commit (to be squashed) that implements the required changes to that `cli.markdown` stays nearly the same as before. _("Everything must change so that everything can stay the same.")_ :-)